api-test/internal/e2e/protocol_test.go

```
@@ -0,0 +1,407 @@
+		}
+	}
+}
+
+// Guards the assumption publicJWK and the DPoP jwk both rely on: a big.Int
+// round-trip of the RSA exponent.
+func TestDPoPJWKExponentRoundTrips(t *testing.T) {
+	d, err := newDPoPSigner("RS256")
+	if err != nil {
+		t.Fatalf("newDPoPSigner: %v", err)
+	}
+	eb, err := b64urlDecode(d.jwk["e"].(string))
```

golangci-lint (2.12.2)

[error] right hand must be only type assertion

(forcetypeassert)